src/MainWindow.cpp

```diff
@@ -231,8 +231,11 @@ void MainWindow::init()
         connect(recentFileActs[i], &QAction::triggered, this, &MainWindow::openRecentFile);
     }
     for(int i = 0; i < MaxRecentFiles; ++i)
-        ui->fileMenu->insertAction(ui->fileExitAction, recentFileActs[i]);
-    recentSeparatorAct = ui->fileMenu->insertSeparator(ui->fileExitAction);
+        ui->fileRecentFiles->insertAction(ui->fileExitAction, recentFileActs[i]);
+
+    QAction *clearRecentFilesAction = ui->fileRecentFiles->addAction(tr("Clear Menu"));
```

Comment:
"Clear List" seems more appropiate, since the menu itself will be kept, but not the list of recent files.

Reply:
I agree with you! I just rename by your opinion. (6bdcb7d6179fa9564fd9e4127d6f09865739fe9d)